RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
+
+        }
+        private string name;
+
+        public string Name
+        {
+            get { return name; }
+            set
+            {
+                if (!value.Equals(""))
+                    name = value;
+                else
+                    throw new ArithmeticException("Illegal name, name must be string");
+            }
+        }
+
+        private int idShelf;
```

Comment:
The correct naming should be shelfId, this is not critical but it is a usually practised standard and grammatically correct.
This is true for all other cases, please switch them